build/Dockerfile

```diff
@@ -1,8 +1,10 @@
 FROM centos:8
-
+ARG NMSTATE_COPR_REPO
+ARG NM_COPR_REPO
 RUN dnf install -b -y dnf-plugins-core && \
-    dnf copr enable -y networkmanager/NetworkManager-1.26 && \
-    dnf copr enable -y nmstate/nmstate-0.3 && \
+    dnf copr enable -y networkmanager/$NM_COPR_REPO && \
+    dnf copr enable -y nmstate/$NMSTATE_COPR_REPO && \
+    dnf copr enable -y nmstate/nispor && \
```

Comment:
nispor needs to have a matching version too, don't they release versioned copr?

Reply:
Nope they don't have versioned copr yet.